docs/recent-releases/go-ipfs-0-6/changelog.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Changelog
```

Comment:
I don't think we should include the changelog in the docs

Reply:
Fair enough. I'll just link to the changelog section on the release page.